hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/common/CachingBlockManager.java

```diff
@@ -433,13 +434,13 @@ private void addToCacheAndRelease(BufferData data, Future<Void> blockFuture) {
     }
 
     try {
-      Await.result(blockFuture);
+      blockFuture.get(); //TODO consider calling get(long timeout, TimeUnit unit) instead
       if (data.stateEqualsOneOf(BufferData.State.DONE)) {
         // There was an error during prefetch.
         return;
       }
     } catch (Exception e) {
-      String message = String.format("error waitng on blockFuture: %s", data);
+      String message = String.format("error waiting on blockFuture: %s", data);
```

Comment:
or move this into the log.error and the {} braces for slf4j to take over, including catching any errors in data.tostring

Reply:
I changed this to a slf4j message with `{}` token